src/inc/caparser.h

```diff
@@ -28,84 +28,21 @@ class CustomAttributeParser {
         LIMITED_METHOD_CONTRACT;
     }
 
-private:
-    signed __int8    GetI1()
-    {
-        LIMITED_METHOD_CONTRACT;
-        signed __int8 tmp = *reinterpret_cast<const signed __int8*>(m_pbCur);
-        m_pbCur += sizeof(signed __int8);
-        return tmp;
-    }
-    unsigned __int8  GetU1()
-    {
-        LIMITED_METHOD_CONTRACT;
-        unsigned __int8 tmp = *reinterpret_cast<const unsigned __int8*>(m_pbCur);
-        m_pbCur += sizeof(unsigned __int8);
-        return tmp;
-    }
-
-    signed __int16   GetI2()
-    {
-        LIMITED_METHOD_CONTRACT;
-        signed __int16 tmp = GET_UNALIGNED_VAL16(m_pbCur);
```

Comment:
Unaligned access means that it accesses address that is not aligned to the size of the data you are accessing. That's why byte access cannot ever be unaligned, as @jkotas has said.